google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectRepositoriesModelItem.java

```diff
@@ -53,26 +50,36 @@ public ProjectRepositoriesModelItem(@NotNull String cloudProject,
     setUserObject(cloudProject);
   }
 
-  public void loadRepositories() {
-    try {
-      ListReposResponse response = cloudRepositoryService.list(user, cloudProject);
-
-      removeAllChildren();
-      List<Repo> repositories = response.getRepos();
-      if (!response.isEmpty() && repositories != null) {
-        repositories.forEach(repo -> {
-          Object name = repo.get("name");
-          if (name != null) {
-            add(new RepositoryModelItem(name.toString()));
+  public void loadRepositories(Runnable onComplete) {
+    cloudRepositoryService
+        .listAsync(user, cloudProject)
+        .thenAccept(response -> {
+          if (response == null) {
```

Comment:
removed the null check. it will either return a response or throw an exception, never null.